protos/flyteidl/admin/launch_plan.proto

```diff
@@ -60,10 +60,8 @@ message LaunchPlanList {
 // Defines permissions associated with executions created by this launch plan spec.
 // Deprecated.
 message Auth {
-    oneof method {
-        string assumable_iam_role = 1;
-        string kubernetes_service_account = 2;
-    }
+    string assumable_iam_role = 1;
```

Comment:
while we are at it... docs?
also maybe elaborate on where each will be used?

Reply:
Yes please

Reply:
done, although technically this message is deprecated 🙃

Reply:
OMG I just saw that... yes yes sorry had you do this...